packages/core/integration-tests/test/integration/glob-package-async/index.js

```diff
@@ -0,0 +1,7 @@
+const scoped = import('@scope/pkg/foo/*.js');
+const unscoped = import('pkg/bar/*.js');
+
+module.exports = async function () {
+    await promise.all([scoped, unscoped]);
+    return scoped.a + scoped.b + unscoped.x + unscoped.y;
```

Comment:
I think it should be `Promise.all`? And each key on the returned object from `import()` is a function that also resolves to a promise, so this should be `await scoped.a() + await scoped.b() ...`